stress_test/controller_test_basic.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2016 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+"""Basic testing stress_test/controller.py"""
+
+import logging
+import json
+import os
+import controller
+import util.netutil
+import util.process
+import sys
+
+#define a root logger
+LOGGER=logging.getLogger()
+LOGGER.level = logging.INFO
+
+# logging output to stdout
+STREAM_HANDLER = logging.StreamHandler(sys.stdout)
+LOGGER.addHandler(STREAM_HANDLER)
+
+
+logging.info('Parsing test configuration')
+
+#define Class inputs:json_conf_file and ctrl_base_dir 
+with open("controller_test.json","r") as json_conf_file:
+    config_file = json.load(json_conf_file)
```

Comment:
better name this var `test_config`, because `config_file` might mislead that this variable refers to a file